packages/typespec-vscode/package.json

```diff
@@ -59,13 +59,24 @@
     ],
     "configuration": [
       {
-        "title": "TypeSpec Language Server Path",
+        "title": "TypeSpec",
         "properties": {
           "typespec.tsp-server.path": {
             "type": "string",
             "default": "",
             "description": "Path to `tsp-server` command that runs the TypeSpec language server.\n\nIf not specified, then `tsp-server` found on PATH is used.\n\nExample (User): /usr/local/bin/tsp-server\nExample (Workspace): ${workspaceFolder}/node_modules/@typespec/compiler",
             "scope": "machine-overridable"
+          },
+          "typespecLanguageServer.trace.server": {
```

Comment:
```suggestion
          "typespec.trace.server": {
```

Don't think this should be another key

Reply:
this needs to be the same as the id of the client for it to work. Do you think it's fine to change the id to "typespec" when starting the client in extension.ts? thanks.
![image](https://github.com/microsoft/typespec/assets/6038235/fb15f92a-9097-44f2-9073-76396fb18b82)

Reply:
Hhm yeah I think it would be better to have a single key, the fact that it is a lanaguage server vs just an extension should be an implementation detail for the user.
What do other extension do?

Reply:
checked some other extensions and they simply set the clientId language. so just update to use "typespec" in the settings and id. thx